components/Nav/NavMobile/NavMobile.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import Link from 'next/link';
+import { donateLink, s3 } from 'common/constants/urls';
+import { navItems } from 'common/constants/navigation';
+import flattenDepth from 'lodash/flattenDepth';
+import HamburgerIcon from 'static/images/icons/hamburger.svg';
+import styles from './NavMobile.css';
+
+export default class NavMobile extends Component {
+  constructor(props) {
```

Comment:
Can you drop the constructor and just use arrow functions for all of the component methods? See below for example.